Source/Blazorise/Components/Table/TableRowCell.razor.cs

```diff
@@ -146,5 +146,15 @@ public Color Color
     /// </summary>
     [Parameter] public RenderFragment ChildContent { get; set; }
 
+    /// <summary>
+    /// Used to prevent the default action for an <see cref="OnClickHandler"/> event.
+    /// </summary>
+    [Parameter] public bool ClickPreventDefault { get; set; }
```

Comment:
Naming seems somewhat weird. But I can't find a better alternative.

Can you just add the new API to the documentation?

Reply:
I personally don't think it does.
Maybe `OnClickPreventDefault`. But I'm fine with either.

Reply:
Oh I guess your concern is that this is kind of a js nomeanclature, the prevent default? I'd just keep it anyway, most front end devs will most likelly interpret it correctly, if we try to come up with a new one it might not be a good idea.

Reply:
Whoever does not understand it, probably has no need for it anyway. hehe